blocks-touch/b-slider/b-slider.js

```diff
@@ -332,6 +330,11 @@
 
             e = e.originalEvent;
 
+            // каскадная подписка на move / up
+            this
+                .bindToWin( 'pointermove', this._onPointerMove )
```

Comment:
Codestyle. You should remove spaces after/before opening/closing parenthesis. Here and below.